test/functional/apps/visualize/_tsvb_chart.js

```diff
@@ -28,28 +28,56 @@ export default function ({ getService, getPageObjects }) {
         .then(function clickMetric() {
           return PageObjects.visualBuilder.clickMetric();
         })
+        .then(function addSeries() {
+          return PageObjects.visualBuilder.addSeries();
+        })
+        .then(function addMetric() {
+          return PageObjects.visualBuilder.addMetric();
+        })
+        .then(function selectOverallAverage() {
+          return PageObjects.visualBuilder.selectItem('.vis_editor__series:last-child ' +
+          '.vis_editor__series-row .ui-sortable-item:last-child .vis_editor__row_item:last-child',
+          'Overall Sum');
+        })
+        .then(function selectCountMetric() {
+          return PageObjects.visualBuilder.selectItem('.vis_editor__series:last-child ' +
+          '.vis_editor__series-row .ui-sortable-item:last-child .vis_editor__std_sibling-metric',
+          'Count');
+        })
         .then(function sleep() {
-          return PageObjects.common.sleep(1003);
+          return PageObjects.common.sleep(1004);
```

Comment:
Why the change from 1003 to 1004?  A ms shouldn't actually make any difference you'd like.

Reply:
Whoops... hit the ol' <C-a> in VIM and it incremented it